### ISSUE TITLE
Paragraph does not make sense

### ISSUE THREAD

**Issue by sebbASF** [apache/infrastructure-asfyaml#3]
The following paragraph does not make sense to me; it does not seem as though the two sentences are related. Or if they are, then some text is missing that explains the link:

"If you would like to add features you are free to open a pull request and propose your changes. The whole logic is defined in the asfyaml.py file that you create."

**Comment by netomi** [comment]
yeah that could be further explained. Otoh, the existing architecture is quite easy to pick up and several people already created PRs to add new features.
